pkg/api/job/models.go

```diff
@@ -5,6 +5,14 @@ import (
 	"github.com/sitehostnz/gosh/pkg/models"
 )
 
+const (
+	// SchedulerType is a job that runs on a scheduler,  I am sure someone can fill this out more.
+	SchedulerType string = "scheduler"
+
+	// DaemonType is a job that runs on a daemon, I am sure someone can fill this out more.
+	DaemonType = "daemon"
+)
+
```

Comment:
These are used in the terraform provider, so we can determine what type of task we are waiting for. Felt like it belonged here since it is something described by the API.